wsf_scraping/pipelines.py

```diff
@@ -22,24 +22,33 @@ def __init__(self):
             self.settings['SECTIONS_KEYWORDS_FILE']
         )
 
+        if not os.path.isdir('./results/pdfs'):
+            os.makedirs('./results/pdfs')
+            os.makedirs('./results/pdfs/nice')
+            os.makedirs('./results/pdfs/who_iris')
```

Comment:
It might worth adding a list of spiders in the settings from which you will create the folders and check that the spider name is in the list when someone is calling process item to avoid the code trying to save in a folder that does not exist.

Reply:
Good idea! I'll do it in the next rev.